aws/README.md

```diff
@@ -6,16 +6,30 @@
 3. [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
 
 ### Set Your Variables
-1. Clone this repository to your local machine.
-2. Uncomment and replace variables in `terraform.tfvars` with the desired values. Check `variables.tf` for default values.
+Uncomment and replace variables in `terraform.tfvars` with the desired values. Check `variables.tf` for default values.
 
 ### Deploy the Cluster
-From inside the cloned directory:
+From inside the `./aws` directory of this project:
 1. Run `terraform init`. This will initialize the terraform project.
 2. Run `terraform plan`. This will perform a dry run and find any problems with your configuration.
 3. Run `terraform apply`. This will deploy the cluster to AWS. Type 'yes' at the prompt.
 
-Congrats! you have a Kubernetes cluster! Next, [it's time to deploy Sourcegraph](https://docs.sourcegrapn.com/admin/deploy/kubernetes/configure)
+It will take a few minutes to deploy the cluster. When finished, you will be greeted with a success message.
+Congratulations! You have a Kubernetes cluster. Next, it's time to [deploy Sourcegraph](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure).
```

Comment:
Remove "Congratulations!" (see previous comment about avoiding overly colloquial/marketing-style language)

Reply:
Pushed this exact change to main.